packages/cli-bot/src/runnable/bot-factory.js

```diff
@@ -1,10 +1,14 @@
 #!/usr/bin/env node
```

Comment:
it's safe to get rid of this file completely - I forgot to delete it since i've moved it to `botkit` of `protocols` repo

Reply:
Done.